### ISSUE TITLE
Error when adding widget in compose mail message 

### ISSUE THREAD

**Issue by haylahi** [0k/web_ckeditor#1]
hello
good job, it's more better than native openerp cleditor,
well, when i try to add ckeditor widget in compose mail message i have the error :
Uncaught The editor instance "body" is already attached to the provided element
thk

**Comment by vaab** [comment]
Hi !

I managed to reproduce your issue. I'm on it.

The form gets actually correctly rendered with `ckeditor` interface, but when hitting "Send", a javascript error pops out with the following message:

```
The editor instance "body" is already attached to the provided element
```

This is due to a spurious `reload` sent after sending the message asking to re-attach `ckeditor` jquery object upon the `body` HTML element. Although there are obvious dirty trick that comes to mind, I'm looking for the best way to avoid this.

**Comment by vaab** [comment]
Could you please confirm the issue is closed for you ?

Thanks for reporting this issue.

**Comment by haylahi** [comment]
Hi,

it's work like charms only in debug mode !!! but when i logout and try to login in normal mode i have another type of error. c'est bizarre :-) 

To reproduce this error, 
1 - edit partner form or other form with web_ckeditor widget in debug mode.(it 's ok)
2- logout and login in normal mode,
Uncaught TypeError: Cannot read property 'options' of undefined
http://localhost:8069/web/webclient/js?mods=web_diagram%2Cweb_ckeditor%2Cbase%2Cbase_setup%2Cprocess%2Cmail%2Cemail_template%2Cbase_calendar%2Cboard%2Ccrm%2Cshare%2Cauth_signup%2Cportal%2Cportal_crm%2Cbase_import%2Cedi%2Caccount%2Cl10n_tn%2Cweb_graph%2Cweb_view_editor%2Cweb_calendar%2Cauth_oauth%2Cauth_oauth_signup%2Cweb_tests%2Cknowledge%2Cdocument%2Cgoogle_docs%2Cweb_gantt%2Caccount_voucher%2Csale%2Cauth_openid%2Cdocument_page:558,

**Comment by vaab** [comment]
Well, this is a new issue. I'm opening a new one. It seems to be CKEDITOR code which somehow does not go well with the concatenation of js files done by openerp.

new issue: https://github.com/0k/web_ckeditor/issues/2

Thank you for your report. This last issue has also been corrected, feel free to confirm this in the #2 issue thread.
